app/models/publisher.rb

```diff
@@ -148,6 +137,20 @@ def inspect_brave_publisher_id
     end
   end
 
+  def publication_type
+    if self.brave_publisher_id.present?
+      :site
+    elsif self.youtube_channel_id.present?
+      :youtube_channel
+    else
+      :unselected
+    end
+  end
+
+  def owner_identifier
+    "oauth#google:#{auth_user_id}"
```

Comment:
Should this return `nil` if auth_user_id is blank? for clarity that it requires oauth.